### ISSUE TITLE
0.0.12  - generated skeleton redundant routes

### ISSUE THREAD

**Issue by necrodome** [jsguy/misojs#20]
Newly generated code still adds the following routes, which makes miso run fail.

```json
{
	"/": { "method": "get", "name": "home", "action": "index" },
	"/user/:user_id": { "method": "get", "name": "user", "action": "edit" }
}
```

**Comment by jsguy** [comment]
Ah, are you using a skeleton in the instance?
What command did you run to make this happen?

**Comment by necrodome** [comment]
I mean the default skeleton: miso -n miso-todo

**Comment by jsguy** [comment]
Argh, my bad - this is now fixed in 0.0.13, can you please see if that works for you?

**Comment by necrodome** [comment]
Yes, 0.0.13 fixes it. Thank you.
